src/components/PopUps/index.css

```diff
@@ -30,6 +30,10 @@
     width: 300px;
 }
 
+.pop-up-message-box-callback-button {
+    margin-top: 10px !important;
```

Comment:
Instead of this, can we change
```
.pop-up-message-box-alert-layout {
...
    display: flex;
    flex-direction: column;
    gap: 10px;
}
```